app/src/main/java/com/mxt/anitrend/model/entity/anilist/meta/FuzzyDate.java

```diff
@@ -75,6 +75,6 @@ public boolean isValidDate() {
 
     @Override
     public String toString() {
-        return String.format(Locale.getDefault(),"%d/%d/%d", year, month, day);
+        return String.format(Locale.getDefault(),"%d/%d/%d", year, Math.max(1, month), Math.max(1, day));
```

Comment:
SimpleDateFormat doesn't like parsing 2021/0/0, it outputs some date mid November 2020...